plugin/trace/setup.go

```diff
@@ -83,6 +83,21 @@ func traceParse(c *caddy.Controller) (*trace, error) {
 				if err != nil {
 					return nil, err
 				}
+			case "datadog_analytics_rate":
```

Comment:
I went with this
I will write the doc and test once we're in agreement over this :)

Reply:
✅  Wrote the doc and added a few test case.